example/Vagrantfile

```
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+ config.vm.box = "ubuntu/xenial64"
```

```suggestion
 config.vm.box = "ubuntu/bionic64"
```

I prefer to use bionic instead of xenial.

---

Quite strange, with bionic64, it stopped working for me when running the `example`. I'm curious if you had to change something on `make run` to get the `backing_store` created outside `/vagrant/example`. That's the only way I could get it working with bionic64. On Xenial64, it works as expected.

---

@zoetrope the only way I could it make the `example` to work with `precise64` on my local ubuntu box was by setting the backing_store to be created in the / disk. As you've mentioned, using the shared Vagrant volume does not work (surprisingly worked for me on xenial64). Would you be okay with changing the backing_store to somewhere else, like `/tmp/topo-build/backing_store`, for example?

---

Thank you for investigating! Hmm, that's strange...

I think it's a good idea to create the `backing_store` outside `/vagrant/example`.